utils/create-jobs.py

```diff
@@ -44,7 +44,7 @@ def main():
         args.job_name += ' - {}'.format(args.job_id)
 
         if args.job_index is not None:
-            args.urlbase = urlparse.urljoin(args.urlbase, args.job_index + '/')
+            args.urlbase = urlparse.urljoin(args.urlbase, args.job_index)
```

Comment:
Are you sure about this ? I remember I had issue with urljoin when the trailing '/' was missing, it discarded the last directory

Reply:
Yes, otherwise you'll get double / in the url but I had the same issue for the first one